frontend/sw360-portlet/src/main/webapp/html/projects/edit.jsp

```diff
@@ -168,6 +170,20 @@ require(['jquery', 'modules/sw360Validate', 'modules/confirm' ], function($, sw3
 
         $('#formSubmit, #formSubmit2').click(
             function () {
+                var valid = true;
+                $.each($('#externalIdsTable tr.bodyRow'), function (i, row) {
+                    var key = $('#'+row.id+' td input.keyClass').val();
+                    var value = $('#'+row.id+' td input.valueClass').val();
+                    var isOneEmpty = ((key !== '') && (value === '')) || ((key === '') && (value !== ''));
+                    if (isOneEmpty) {
+                        $.alert('There is a blank field in one or more of your external ids. Please correct this before updating the project.');
+                        valid = false;
+                        return false;
```

Comment:
duplicated code!

Reply:
js removed and added required/minlength to the external id input fields